### ISSUE TITLE
loss error when training my dataset in faster rcnn-resnet50

### ISSUE THREAD

**Issue by Smoothing97** [open-mmlab/mmdetection#510]
I have a bad result while training:

- in line1,the acc: 3.5156
- loss_rpn_cls is increasing
- loss_cls>1,loss>1

I'm training faster_rcnn_r50_fpn_1x.py on DOTA dataset using one GTX1080ti.the size of origin images are 1000*1000-2000*2000.When I cut them into 1024*1024,the memory is not enough.So I cut them into 256*256 and set:
imgs_per_gpu=32,
workers_per_gpu=2,
but all of the loss are nan.
I change the set as:
imgs_per_gpu=1,
workers_per_gpu=1,
which I find:
2019-04-17 15:27:05,922 - INFO - Epoch [1][1/48707]     lr: 0.00000, eta: 49 days, 22:58:06, time: 7.385, data_time: 4.735, loss_rpn_cls: 0.6962, loss_rpn_reg: 0.0047, loss_cls: 2.7209, acc: 3.5156, loss_reg: 0.0261, loss: 3.4479
2019-04-17 15:27:06,139 - INFO - Epoch [1][2/48707]     lr: 0.00000, eta: 25 days, 17:03:34, time: 0.217, data_time: 0.003, loss_rpn_cls: 0.6958, loss_rpn_reg: 0.0000, loss_cls: 2.8554, acc: 0.9766, loss_reg: 0.0000, loss: 3.5512
2019-04-17 15:27:06,310 - INFO - Epoch [1][3/48707]     lr: 0.00000, eta: 17 days, 12:39:56, time: 0.172, data_time: 0.003, loss_rpn_cls: 0.7001, loss_rpn_reg: 0.0021, loss_cls: 2.6812, acc: 3.3203, loss_reg: 0.0000, loss: 3.3834
2019-04-17 15:27:06,488 - INFO - Epoch [1][4/48707]     lr: 0.00000, eta: 13 days, 10:41:36, time: 0.177, data_time: 0.004, loss_rpn_cls: 0.6936, loss_rpn_reg: 0.1015, loss_cls: 2.6756, acc: 2.1484, loss_reg: 0.0951, loss: 3.5659
2019-04-17 15:27:06,646 - INFO - Epoch [1][5/48707]     lr: 0.00000, eta: 10 days, 23:17:19, time: 0.158, data_time: 0.002, loss_rpn_cls: 0.6901, loss_rpn_reg: 0.0442, loss_cls: 2.6899, acc: 0.3906, loss_reg: 0.0230, loss: 3.4471
2019-04-17 15:27:06,807 - INFO - Epoch [1][6/48707]     lr: 0.00000, eta: 9 days, 7:46:31, time: 0.161, data_time: 0.003, loss_rpn_cls: 0.6985, loss_rpn_reg: 0.1654, loss_cls: 2.7227, acc: 4.1016, loss_reg: 0.0175, loss: 3.6042
2019-04-17 15:27:07,017 - INFO - Epoch [1][7/48707]     lr: 0.00000, eta: 8 days, 4:40:22, time: 0.210, data_time: 0.003, loss_rpn_cls: 0.6938, loss_rpn_reg: 0.0035, loss_cls: 2.8073, acc: 0.9766, loss_reg: 0.0246, loss: 3.5291
2019-04-17 15:27:07,205 - INFO - Epoch [1][8/48707]     lr: 0.00000, eta: 7 days, 7:54:42, time: 0.188, data_time: 0.003, loss_rpn_cls: 0.6933, loss_rpn_reg: 0.0347, loss_cls: 2.6979, acc: 1.9531, loss_reg: 0.0794, loss: 3.5053
2019-04-17 15:27:07,533 - INFO - Epoch [1][9/48707]     lr: 0.00000, eta: 6 days, 18:16:27, time: 0.328, data_time: 0.003, loss_rpn_cls: 0.6996, loss_rpn_reg: 0.2419, loss_cls: 2.5427, acc: 5.4688, loss_reg: 0.0689, loss: 3.5531
2019-04-17 15:27:07,662 - INFO - Epoch [1][10/48707]    lr: 0.00000, eta: 6 days, 4:08:15, time: 0.129, data_time: 0.003, loss_rpn_cls: 0.7088, loss_rpn_reg: 0.0653, loss_cls: 2.5281, acc: 21.6797, loss_reg: nan, loss: nan
2019-04-17 15:27:08,643 - INFO - Epoch [1][11/48707]    lr: 0.00000, eta: 6 days, 5:09:25, time: 0.982, data_time: 0.003, loss_rpn_cls: nan, loss_rpn_reg: nan, loss_cls: nan, acc: 83.3333, loss_reg: nan, loss: nan
2019-04-17 15:27:09,582 - INFO - Epoch [1][12/48707]    lr: 0.00000, eta: 6 days, 5:25:27, time: 0.938, data_time: 0.003, loss_rpn_cls: nan, loss_rpn_reg: nan, loss_cls: nan, acc: 71.4286, loss_reg: nan, loss: nan

I'm reading the source code and try to find the reason.Really need some help,thanks！

**Comment by tsvetiko** [comment]
Looking at your learning rate, it looks way too small. You need to adjust it based on the number of GPUs and batch size you're using following the Linear Scaling Rule described [here](https://github.com/open-mmlab/mmdetection/blob/master/docs/GETTING_STARTED.md#train-a-model). So let's say you're training with a single GPU and the flag `imgs_per_gpu` in the original config was set to `2`, then you need to set it to `8*2=16`, because the default learning rate was set for 8 GPUs. If you don't want to change the `imgs_per_gpu`, then you need to change the `lr` in the `optimizer` flag in your config.

NOTE:
If you get `NaN` from the very beginning when you start training, then something is wrong with your data. So you need to check if your images, labels and bounding boxes are correct. Keep in mind that the `bbox` should be in the following format: `[ x_min, y_min, x_max, y_max]`, meaning [top, left, bottom, right].
